src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,46 @@
+.calendar {
+  display: flex;
+  flex-flow: row wrap;
+  gap: $gap;
+  width: $calender-width;
+  margin: 0 auto;
+  padding: 30px $calendar-padding;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    @include square($day-size);
+    background-color: $day-background-color;
+    border: 1px solid black;
+    transition-duration: 500ms;
```

Comment:
also, define transition-properties (list all properties you change) - it's better for performance than the default `all` value